rfc/github-cdevents.md

```diff
@@ -0,0 +1,58 @@
+| dev.cdevents.repository.imported | repository_import           | when a new repository is imported to Github. (should be defined in the spec)                                                                                                                                                                                                                                                                                                                                                                                      |
+| dev.cdevents.repository.*        | repository                  | This event occurs when there is activity relating to repositories. <details><summary>Actions</summary> {"Action type":{"created", "modified", "deleted", "edited", "privatized", "publicized", "renamed", "transfered", "unarchived", "archived"}</br>                                                                                                                                                                                                            |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
```

Comment:
We should map these to CDEvents that exist where possible:

`created`: [created](https://github.com/cdevents/spec/blob/main/source-code-version-control.md#repository-created)
`modified`, `edited`, `privatized`, `publicized`:  [modified](https://github.com/cdevents/spec/blob/main/source-code-version-control.md#repository-modified)
`deleted`: [deleted](https://github.com/cdevents/spec/blob/main/source-code-version-control.md#repository-deleted)

`renamed` could be a flavour `modified` as well, but it could be considered as a `deleted` + `created` as well. 
`transferred` poses similar questions too, we don't have a concept of `org` in CDEvents (yet), so it would be similar to a `rename` I think.

`unarchived`, `archived` are probably flavours of `modified` too.